packages/common/src/store/premium-content/sagas.ts

```diff
@@ -233,8 +238,9 @@ function* handleSpecialAccessTrackSubscriptions(tracks: Track[]) {
     }
 
     const hasNoSignature = !premiumContentSignature
-    const isFollowGated = !!premiumConditions?.follow_user_id
-    const isTipGated = !!premiumConditions?.tip_user_id
+    const isFollowGated =
+      premiumConditions.type === PremiumContentType.FOLLOW_GATED
```

Comment:
should this line and the one below it have the ? i.e. `premiumConditions?.type === PremiumContentType.FOLLOW_GATED` ?